### PR TITLE
RN update mods to use RP instead of TR

### PR DESCRIPTION
**Pull request by raidernick**
-Update my mods to use reflection plugin instead of texture replacer.

### CODE REVIEW

**Comment by Olympic1** [comment]
#rebuild

File: NetKAN/RN-SovietRockets.netkan [hunk 1]
```diff
@@ -23,7 +23,7 @@
     [
 		{ "name"    : "BDAnimationModules" },
 		{ "name" 	  : "ModuleManager" },
-        { "name"    : "TextureReplacer" }
+        { "name"    : "ReflectionPlugin" }
```
Comment (Olympic1) [inline NetKAN/RN-SovietRockets.netkan]: @raidernick RNSovietRockets depends on the ReflectionPlugin but also conflicts with it (see line 20). Could you fix this?

**Comment by raidernick** [comment]
done

**Comment by Olympic1** [comment]
Thanks, merging

**Pull request merged**